components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.management.apis/src/main/java/org/wso2/micro/integrator/management/apis/UserResource.java

```diff
@@ -163,7 +164,7 @@ protected JSONObject handleDelete(MessageContext messageContext)
         UserStoreManager userStoreManager = Utils.getUserStore(domain);
         String[] roles = userStoreManager.getRoleListOfUser(user);
 
-        if (ADMIN.equals(performedBy)) {
+        if (this.superAdminUsername.equals(performedBy)) {
```

Comment:
Shall we check the possibility of using [UserCoreUtil.isPrimaryAdminUser()](https://github.com/wso2/product-micro-integrator/blob/master/components/org.wso2.micro.integrator.security/src/main/java/org/wso2/micro/integrator/security/user/core/util/UserCoreUtil.java#L660) ?